### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by lucashervier**
# Issues template

Provide a template for issues according to #89  issue

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request/Enhancement suggestion
+about: Propose new features / Suggest an idea, modification for this project
```
Comment (justinplakoo) [inline .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md:3]: We could add:
```
Select the modules to which the bug refers:

* [ ] Attributions Methods
* [ ] Feature Visualization
* [ ] Concepts
* [ ] Metrics 
* [ ] Documentation 
```

File: .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: "[Enhancement] - Your suggestion/feature request"
```
Comment (justinplakoo) [inline .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md:4]: We could add:
```
Select the modules to which the bug refers:

* [ ] Attributions Methods
* [ ] Feature Visualization
* [ ] Concepts
* [ ] Metrics 
* [ ] Documentation 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] - Your issue"
```
Comment (justinplakoo) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: We could add:
```
Select the modules to which the bug refers:

* [ ] Attributions Methods
* [ ] Feature Visualization
* [ ] Concepts
* [ ] Metrics 
* [ ] Documentation 
```

Comment (lucashervier) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: Done

**Review by justinplakoo — APPROVED** [review]

**Comment by Holt59** [comment]
You might want to check issue form instead of issue templates, it's quite new (maybe still in beta?) but I guess it's going to take over issue templates soon - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

File: .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (fel-thomas) [inline .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md:11]: Haha super good idea, I like that !

File: .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+**Is this issue just a suggestion or a modification you would like to see ? Please describe.**
+A clear and concise description of your suggestion. Ex. It would be nice to be able to [...]
```
Comment (fel-thomas) [inline .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md:14]: e.g. ?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (fel-thomas) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: e.g

**Review by fel-thomas — COMMENTED** [review]
This is a great idea and a great plug to start with, gg Lucas! 
I think @Holt59  is right (like always), we should also keep an eye on the form issue...

File: .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md (hunk 5 above)
Comment (fel-thomas) [reply .github/ISSUE_TEMPLATE/feature-request-enhancement-suggestion.md:14]: or i.e

**Review by fel-thomas — APPROVED** [review]
All my comments are adressed. LGTM !

**Pull request merged**